doc/release-notes/template.sgml

```diff
@@ -0,0 +1,117 @@
+<!doctype linuxdoc system>
+<article>
+<title>Squid @SQUID_VERSION@ release notes</title>
+<author>Squid Developers</author>
+
+<abstract>
+This document contains the release notes for version @SQUID_RELEASE@ of Squid.
+Squid is a WWW Cache application developed by the National Laboratory
+for Applied Network Research and members of the Web Caching community.
+</abstract>
```

Comment:
Feels wrong to add a template with such outdated information. Do you plan to update it shortly after merging this?

Reply:
Not without board approval. This particular text is related to the copyright and ownership legacy.

Reply:
I disagree that Board approval is needed for this change. This change does not affect Squid copyright or ownership. The legacy issues have been already resolved (e.g., see 2013 commit b1427da).

@kinkie, would you be against the removal of stale/misplaced information about the National Laboratory for Applied Network Research from the release file template, given the facts that NLANR (as an informal organization) could not hold any copyright, does not participate in any modern Squid releases, and no longer exists? If you and Amos support removing that info, then we have Board approval regardless of whether it is actually needed or not.

Reply:
I agree that if NLANR doesn't hold any title, there is no need to mention that except as an acknowledgement for the support given and leadership shown

Reply:
@yadij, AFAICT, Board support is not needed but, if you yourself support this removal, then that Board support does exist. In other words, the Board is _not_ in the way of this PR removing that stale text.

> except as an acknowledgement for the support given and leadership shown

Surely, v7+ release notes are not the right place for acknowledging good deeds that happened many releases (and years) ago!

Reply:
This statement is the only record in Squid sources that NLANR was involved with Squid development.

@rousskov, does Duane or Measurement Factory have records of which Squid versions were done under their grants?
I could accept moving them to the SPONSORS listing for those versions.

Reply:
> This statement is the only record in Squid sources that NLANR was involved with Squid development.

This assertion is inaccurate because there are many other mentions of NLANR, including many release notes and the Programming Guide.

And even if that assertion were accurate, one can still address my change request without dropping this "record", by placing a reference to NLANR somewhere else. The essence of this change request is that the proposed text does not belong to future release notes and, hence, should not be in this proposed template. We should not be arguing about it.

> does Duane or Measurement Factory have records of which Squid versions were done under their grants?

I cannot speak for Duane, but, AFAIK, Measurement Factory never mapped grants to Squid versions. More importantly, we already reflect those grants in older versions. This change request is about future release notes --  not something even remotely related to NLANR!

Reply:
>  This change request is ... not something even remotely related to NLANR!

I really hope you do not meant that. The Squid code would not be what it is without the past contributors and sponsors.

Reply:
> Alex: This change request is about future release notes -- not something even remotely related to NLANR!

> Amos: I really hope you do not meant that.

I did mean what I said: Future release notes is not the right place to acknowledge specific ancient contributions, no mater how large/valuable/important.

> The Squid code would not be what it is without the past contributors and sponsors.

Very true, but I see no connection between the above fact and my change request (or its rationale).  NLANR contribution  was as huge/valuable/important as they get, but this change request is not about that.

Reply:
> > Alex: This change request is about future release notes -- not something even remotely related to NLANR!
> 
> > Amos: I really hope you do not meant that.
> 
> I did mean what I said: Future release notes is not the right place to acknowledge specific ancient contributions, no mater how large/valuable/important.

*In general* you have (and I agree with the principle). However, the *specific change* context of this discussion is: erasing the acknowledgement of NLANR. Such erasure does not meet that general principle. Thus my approach of moving it to `SPONSORS.list` (which is ordered by latest version of sponsor contribution).

> 
> > The Squid code would not be what it is without the past contributors and sponsors.
> 
> Very true, but I see no connection between the above fact and my change request (or its rationale). NLANR contribution was as huge/valuable/important as they get, but this change request is not about that.

Confusion. This discussion is all about some text naming NLANR. AFAICS, we have agreed that it should not be in release notes and need to agree on where it gets moved to.

Reply:
> However, the _specific change_ context of this discussion is: erasing the acknowledgement of NLANR. 

That is a very misleading summary IMO! The official repository does not have this file. Thus, this change request is not asking to erase something from the official repository! This specific change request is about removing problematic text from a _new_ proposed file. One should view this change request as a request not to add a misplaced acknowledgement of NLANR to the official repository.


> Such erasure does not meet that general principle.

From the official repository point of view (i.e. from the final/merged commit point of view, and that point of view is what matters here), there is no erasure at all.

> Thus my approach of moving it to `SPONSORS.list` (which is ordered by latest version of sponsor contribution).

This change request does not preclude updating SPONSORS.list in any way! If you want to update SPONSORS.list, please update it (or let me know if you want me to update it instead).
 
> > > The Squid code would not be what it is without the past contributors and sponsors.

> > Very true, but I see no connection between the above fact and my change request (or its rationale). NLANR contribution was as huge/valuable/important as they get, but this change request is not about that.
 
> Confusion. This discussion is all about some text naming NLANR. AFAICS, we have agreed that it should not be in release notes and need to agree on where it gets moved to.

Sorry, I do not know what confuses you; FWIW, I see no contradiction or conflict in that exchange, and your response does not identify it AFAICT, so I cannot resolve that confusion directly. Hopefully, my other comments do that indirectly.

I am glad we agree that `doc/release-notes/template.*` should not add this text about NLANR to the official repository. That is all that matters for this specific change request. Again, I am not against adding some text about NLANR somewhere else (e.g., to the blob about NSF grants already present in SPONSORS.list), in this or another PR, before you address this change request or after. You are in the driver's seat for both changes! If you want me to make them, please let me know.

Reply:
[c8de810](https://github.com/squid-cache/squid/pull/1264/commits/c8de8109136bdb4723fb3aed74e60d56c7b4aeea) should resolve this.